src/modules/commander/PreflightCheck.cpp

```diff
@@ -290,15 +290,15 @@ static bool accelerometerCheck(orb_advert_t *mavlink_log_pub, unsigned instance,
 
 			if (accel_magnitude < 4.0f || accel_magnitude > 15.0f /* m/s^2 */) {
 				if (report_fail) {
-					mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: ACCEL RANGE, hold still on arming");
+					mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: ACCEL RANGE, hold still on arming.");
```

Comment:
Change no longer needed. Text already improved in this more recent PR: 
https://github.com/PX4/Firmware/pull/10586/files#diff-a2bcb238e25d8a97ab2aee48f54dbc48R274